Purchases/SubscriberAttributes/RCSubscriberAttributesManager.m

```diff
@@ -115,15 +115,15 @@ - (void)markAttributesAsSynced:(RCSubscriberAttributeDict)syncedAttributes
     }
 }
 
-- (void)storeAttributeLocallyIfNeededWithKey:(NSString *)key value:(NSString *)value appUserID:(NSString *)appUserID {
+- (void)storeAttributeLocallyIfNeededWithKey:(NSString *)key value:(nullable NSString *)value appUserID:(NSString *)appUserID {
     NSString *valueOrEmpty = value ?: @"";
     NSString * _Nullable currentValue = [self currentValueForAttributeWithKey:key appUserID:appUserID];
     if (!currentValue || ![currentValue isEqualToString:valueOrEmpty]) {
         [self storeAttributeLocallyWithKey:key value:value appUserID:appUserID];
     }
 }
 
-- (void)storeAttributeLocallyWithKey:(NSString *)key value:(NSString *)value appUserID:(NSString *)appUserID {
+- (void)storeAttributeLocallyWithKey:(NSString *)key value:(nullable NSString *)value appUserID:(NSString *)appUserID {
```

Comment:
this shouldn't be needed if we make sure that value here is never nil, per my previous comment

Reply:
Removed after applying the change to use valueOrEmpty.